instrumentation/action_pack/test/opentelemetry/instrumentation/action_pack/patches/action_controller/metal_test.rb

```diff
@@ -85,117 +83,13 @@
     _(span.name).must_equal 'HTTP GET'
   end
 
-  describe 'when the application has span_naming set with controller_action' do
-    before do
-      OpenTelemetry::Instrumentation::ActionPack::Instrumentation.instance.config[:span_naming] = :controller_action
-    end
-
-    after do
-      OpenTelemetry::Instrumentation::ActionPack::Instrumentation.instance.config[:span_naming] = nil
-    end
-
-    it 'sets the span name to the HTTP method and route' do
-      get '/ok'
-
-      _(span.name).must_equal 'ExampleController#ok'
-    end
-
-    it 'sets the http.route attribute' do
-      get '/ok'
-
-      _(span.attributes['http.route']).must_equal '/ok(.:format)'
-    end
-
-    it 'sets the span name when the controller raises an exception' do
```

Comment:
These basic scenarios are basically copy/pasta from the above, which is why it's ok to remove.